__tests__/app.test.js

```diff
@@ -209,3 +209,44 @@ describe('GET/api/articles', () => {
         })
     })
 })
+describe('GET/api/articles/:article_id/comments', () => {
+    test('200 status with an array of comment objects that contain appropriate properties', () => {
+        return request(app)
+        .get(`/api/articles/3/comments`)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.newComments).toBeInstanceOf(Array)
+            expect(body.newComments).toHaveLength(2)
```

Comment:
Why are we sending this back on a key of newComments? This implies that we have added something wheras we are simply GETting the exisitng comments